packages/debugger/app/page.tsx

```diff
@@ -30,10 +30,18 @@ export default function App({
   searchParams: Record<string, string>;
 }): JSX.Element {
   const router = useRouter();
-  const url = searchParams.url ?? null;
-  const [urlInput, setUrlInput] = useState(
-    url || process.env.NEXT_PUBLIC_HOST || "http://localhost:3000"
-  );
+  const url = useMemo(() => {
+    const fallbackURL = process.env.NEXT_PUBLIC_HOST || 'http://localhost:3000';
```

Comment:
Usually a client side exception occurs when a server isn't running - does this handle that?

Reply:
This doesn't fix an issue when frames return 500. Now I found out that when the URL is not alive our debugger returns 500 response which causes the debugger to fails as well with the following.

<img width="1927" alt="image" src="https://github.com/framesjs/frames.js/assets/174716/49c733a6-9a00-40db-a816-edf0b61c20e8">

Reply:
<img width="1306" alt="image" src="https://github.com/framesjs/frames.js/assets/174716/b7d34a55-3cf6-4b49-840c-e0f9e89cba95">

Reply:
Yeah it causes an error when you try to debug a url that isn't online so you shouldn't automatically somewhere

Reply:
I think we just need to properly show an error which we do now after the fix.

Reply:
i think it should be possible to visit the root page without specifying a url or having it load an unavailable url automatically

Reply:
Not sure what you mean. So the initial load should not automatically load the URL?

Reply:
But then we should probably not fill the URL input with default URL as well right?

Reply:
i think it's useful to pre-populate the input, no?

Reply:
ok so if the url query is filled, we should load it automatically. If there is no url query param, then we should just show topbar, right?

Reply:
yes sounds good